src/Invictus.Testing/LogicAuthentication.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Threading.Tasks;
+using GuardNet;
+using Microsoft.Azure.Management.Logic;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.Rest;
+
+namespace Invictus.Testing 
+{
+    /// <summary>
+    /// Authentication representation to authenticate with logic apps running on Azure.
+    /// </summary>
+    public class LogicAuthentication
```

Comment:
Question - Should we integrate `ISecretProvider` in here?

Reply:
Yes, but we should re-think our security authentication in the security repo so we can reuse it here, since the authentication models there return a `IKeyVaultClient`.

Reply:
Good for another issue.

Reply:
#34